subprojects/files/files.gradle.kts

```diff
@@ -36,6 +36,9 @@ dependencies {
     implementation(library("commons_lang"))
 
     testImplementation(project(":internalTesting"))
+
+    val testFixturesImplementation by configurations
```

Comment:
This line should be unnecessary. Just remove it.

Reply:
It doesn't work without it :/

Reply:
Strange, I tried it on your branch and it did work for me...

Reply:
I checked, and it works from the command-line indeed, but IntelliJ complains:

![image](https://user-images.githubusercontent.com/495366/46590539-5b54ac80-cab4-11e8-93e9-5d59d57f945c.png)

Reply:
I opted to make the change; we can fix this somewhere else to make IntelliJ understand.

Reply:
Can you make inform the Kotlin DSL team? I think highlighting worked for me...

Reply:
Sent it their way.